css/modules.css

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
Apply the changes to this file that we talked about in class. Namely defining your module styles as classes rather than elements. Be sure to push to "modular-design" branch and __not__ "layout" branch. I would like to keep work from each assignment separate.